### ISSUE TITLE
How to construct a graph

### ISSUE THREAD

**Issue by wj-gxy** [CRIPAC-DIG/GET#1]
Hello, in your repository, I can't find a way to build a graph, please help me point out the way to build a graph, I want to use this as the beginning of my learning, I would be grateful and look forward to your reply.

**Comment by Jason98Xu** [comment]
Hi,
The code is in the file interaction.py, where the graph construction is realized in the class _ClassificationInteractions_. Please take a look at it. If you have any further question, feel free to contact us again.

**Comment by wj-gxy** [comment]
> 

Maybe because I just started to learn the relevant content of the graph, there are a lot of codes that I can't understand. Do you have the code of this code notebook, or do you have any recommended content, thank you very much.

**Comment by Jason98Xu** [comment]
As a starter, I would recommend you to read some representative paper in the field of graph neural networks, such as graph convolutional networks, graph attention networks, and graph gated neural networks, to take a overview of how to propagate information on graphs. There are also lots of blogs about the realization of these GNNs, I think these will be useful for you to get a quick start.

**Comment by wj-gxy** [comment]
Yes, I have read these kinds of articles, and I have also run through the source code, but their datasets are all constructed graphs, including the code of TextGCN, BiGCN papers, I have also run, see your paper is by The text is transferred to the graph, and I want to add knowledge in this area.
